cnr-ispc/.htaccess

```diff
@@ -3,5 +3,9 @@
 Header set Access-Control-Allow-Origin *
 Options +FollowSymLinks
 RewriteEngine on
- 
-RewriteRule ^ontology/branco$ https://dhilab-fi.github.io/branco/current/branco.owl [R=303,L]
\ No newline at end of file
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} html
+RewriteRule ^ontology/branco$ https://dhilab-fi.github.io/branco/current/branco.html [R=303,L]
+
+RewriteRule ^ontology/branco$ https://dhilab-fi.github.io/branco/current/branco.owl [R=303,L]
```

Comment:
This will be a default response, but you won't return the right type in the response. Have a look at https://github.com/perma-id/w3id.org/blob/master/example/.htaccess for a better implementation

Reply:
Are you suggesting the `AddType` setup?  Those are in *lots* of files for some reason.  But for all redirects this server is returning simple HTML and the target server needs to setup file type mappings.  (Unless I've been confused this entire time!)

Reply:
it's also adding rdf/xml. It may be useful to return that in the type as well.